pkg/autoscaler/config/config.go

```diff
@@ -111,6 +111,17 @@ func NewConfigFromMap(data map[string]string) (*autoscalerconfig.Config, error)
 		lc.TargetUtilization /= 100.0
 	}
 
+	// only one of the following should be set, since target utilization have default value,
+	// it is greater than zero regardless of whether set or not
+	if _, isTargetDefined := data["target-utilization"]; isTargetDefined && lc.ContainerConcurrencyTargetFraction > 0 {
+		return nil, fmt.Errorf("target-utilization and container-concurrency-target-percentage are mutually exclusive, please remove container-concurrency-target-percentage")
+	}
+
+	// if only ContainerConcurrencyTargetFraction is set, then set the target utilization to its value
+	if lc.ContainerConcurrencyTargetFraction > 0 {
+		lc.TargetUtilization = lc.ContainerConcurrencyTargetFraction
+	}
```

Comment:
nit: let's add a note to remove these once target % is removed

Reply:
Done.
If `container-concurrency-target-percentage` is kept in config map and just remove the code about `ContainerConcurrencyTargetFraction`, these lines should be kept. If `container-concurrency-target-percentage` is not supported in config map, too. these could be deleted then.